config/routes.rb

```diff
@@ -30,6 +30,8 @@
         format: :xml
 
   post '/api/verify/images' => 'idv/image_uploads#create'
+  post '/api/verify/documents' => 'idv/verify_documents#create'
+  get '/api/verify/documents/:document_capture_session_uuid' => 'idv/verify_documents#index'
```

Comment:
with an ID in the URL I think this is better fitted as a `#show` rather than an `#index`